rllib/offline/offline_data.py

```diff
@@ -30,6 +30,38 @@ def __init__(self, config: AlgorithmConfig):
         self.data_read_method_kwargs = (
             self.default_read_method_kwargs | config.input_read_method_kwargs
         )
+
+        # If a specific filesystem is given, set it up. Note, this could
+        # be `gcsfs` for GCS, `pyarrow` for S3 or `adlfs` for Azure Blob Storage.
+        # this filesystem is specifically needed, if a session has to be created
+        # with the cloud provider.
+        if self.filesystem:
+            if self.filesystem == "gcs":
```

Comment:
nit nit: Should we do:
```
if self.filesystem == "gcs":
   ...
elif self.filesystem is not None:
  raise ...
```

one less indentation.

Reply:
Let's do it. I change it now. Great catch!